sklearn/ensemble/_bagging.py

```diff
@@ -558,6 +577,12 @@ class BaggingClassifier(ClassifierMixin, BaseBagging):
     verbose : int, default=0
         Controls the verbosity when fitting and predicting.
 
+    base_estimator : object, default="deprecated"
+        Use `estimator` instead.
+        .. deprecated:: 1.1
```

Comment:
```suggestion
        Use `estimator` instead.

        .. deprecated:: 1.1
```
The blank line will remove the sphinx warning... same comment in the Regressor

Reply:
Thanks for the review and the feedback @cmarmo ! I'll work on reverting the changes done to `BaseBagging` and further modifying both Bagging classes. I'll also apply your suggestion for fixing the sphinx warnings. 😁

Reply:
This one should still be fixed... :)